elements/morley-wang-xu.def

```diff
@@ -1,5 +1,6 @@
 name: Morley-Wang-Xu
 html-name: Morley&ndash;Wang&ndash;Xu
+notes: A Morley-Wang-Xu element of order \(k\) and reference element dimension \(m\) only includes degrees of freedom on subentities of dimensions \((k - i), 1 \leqslant k \leqslant m \).
```

Comment:
```suggestion
notes:
  - A Morley-Wang-Xu element of order \(k\) and reference element dimension \(m\) only includes degrees of freedom on subentities of dimensions \((k - i), 1 \leqslant k \leqslant m \).
```

Reply:
Need the `-` otherwise the builder loops over each character in the note and does this: 
![image](https://github.com/user-attachments/assets/ccf646ec-7cda-46d0-a7d2-c0c5159548dd)